.evergreen/install-npm-deps.sh

```diff
@@ -31,7 +31,5 @@ npm run mark-ci-required-optional-dependencies
 # along with its types, but npm wouldn't try and compile the addon
 (npm ci && test -e node_modules/mongodb-client-encryption) || npm ci --ignore-scripts
 
-npm run evergreen-release bump
```

Comment:
unless I am misunderstanding something, I don't think this bump was necessary to get the dir-based packages installed for the sake of i.e. tests, etc..
It does seem like that this acted as a preliminary step for different CI runs but the idea going forward is that the repository will have the latest published version and only get bumped before releases and persist for future runs

Reply:
I will double check this as I did see some weird error with browser-repl that may be caused by this